fix(ios): add missing SpeakCore import — unblocks TestFlight releases

## Problem

All iOS TestFlight releases have been failing since March 26 with:

```
SpeakiOSApp.swift:93:46: error: cannot find type 'OpenClawClient' in scope
```

## Root Cause

`SpeakiOSApp.swift` references `OpenClawClient.Conversation` (line 93, defined in `SpeakCore`) but only imports `SpeakiOSLib`, which does not re-export `SpeakCore` via `@_exported import`.

## Fix

Add `import SpeakCore` to `SpeakiOSApp/SpeakiOSApp.swift` (line 2).

## Impact

One-line fix. Unblocks all iOS TestFlight releases. Zero risk.

--- a/SpeakiOSApp/SpeakiOSApp.swift
+++ b/SpeakiOSApp/SpeakiOSApp.swift
@@ -1,4 +1,5 @@
 import SwiftUI
+import SpeakCore
 import SpeakiOSLib
 import UIKit
 

